library/core/src/marker.rs

```diff
@@ -315,6 +315,18 @@ pub trait StructuralEq {
 /// the trait `Copy` may not be implemented for this type; field `points` does not implement `Copy`
 /// ```
 ///
+/// Shared references (`&T`) are also `Copy`, so a type can be `Copy`, even when it holds
+/// shared references of types `T` that are *not* `Copy`. Consider the following struct,
+/// which can implement `Copy`, because it only holds a *shared reference* to our non-`Copy`
+/// type `PointList` from above:
+/// ```
+/// # #![allow(dead_code)]
+/// # struct PointList;
+/// struct PointListWrapper<'a> {
```

Comment:
Adding the `derive` would be nice: since examples are compiled and run, showing it works by implementing it could be nice.

```suggestion
/// #[derive(Copy, Clone)]
/// struct PointListWrapper<'a> {
```

Reply:
That is a good idea, thanks for the suggestion! :+1: 
Do you think we should simultaneously add the same `derive` to the [`Point` struct in line 294](https://github.com/rust-lang/rust/blob/009551f758d1d007ad0f7b652bfa8ddba0738117/library/core/src/marker.rs#L294) for consistency?

Reply:
@poliorcetics with the two latest commits this is now resolved.